src/js/load-package.ts

```diff
@@ -94,15 +94,27 @@ type PackageLoadMetadata = {
   depends: string[];
   done: ResolvablePromise;
   installPromise?: Promise<void>;
+  packageData?: PackageData;
 };
 
+export type PackageType =
+  | "package"
+  | "cpython_module"
+  | "shared_library"
+  | "static_library";
+
 // Package data inside pyodide-lock.json
 export type PackageData = {
```

Comment:
@ryanking13 Do we want to expose all this stuff? What are the stability guarantees we want to make? Maybe we should put a stability tag in the documentation to say that the `PackageData` return type is subject to change? Or mark specific fields as unstable?

A bunch of extra tsdoc tags work now for things like stability e.g., 
https://tsdoc.org/pages/tags/experimental/
or something like that. I'm not sure what the best way to say "this is more likely to change than most of our API surface" is.

Reply:
I think we would like only to expose

- name
- version
- file_name
- package_type

and add experiemntal_tag in `package_type`. Other keys don't need to be exposed.

Reply:
And ideally also filter the values at runtime...